sgf-viewer/src/components/Citation.svelte

```diff
@@ -22,11 +22,11 @@
 }`}</pre>
 </code>
 <code on:click={tdFocus}>
-    <pre>{`@misc{tseng2024challenges,
+    <pre>{`@misc{tseng2024ais,
```

Comment:
The citation blurb matches what you get when you click Export BibTeX Citation on https://arxiv.org/abs/2406.12843